po/fr.po

```diff
@@ -2728,16 +2742,16 @@ msgstr ""
 
 #: /tmp/fish/implicit/share/config.fish:1
 msgid "Update PATH when fish_user_paths changes"
-msgstr ""
+msgstr "Mettre à jour PATH lorsque fish_user_paths est modifiée"
 
 #: /tmp/fish/implicit/share/config.fish:2
 msgid "Signal handler for the TRAP signal. Launches a debug prompt."
 msgstr ""
+"Gestionnaire de signal pour le signal TRAP. Lance une invite de débogage."
```

Comment:
Inconsistent translation of bug / debugging: further up, anglicised versions were used (“bug” and “débugger”) while this line prefers the idiomatic French version (“débogage”, from “bogue”).

Reply:
I align my translation, but it should be corrected later in all translations, as « débogage » seems more widely used.

Reply:
Besides, the translations I didn't modified are not uniformized either.

Reply:
I agree with you about the use of _débogage / bogue_; this comment only goes the other way because I went through your PR from the top down, so I only came to that conclusion a bit later. As consistency of terminology is an improvement in itself, why not change all remaining instances of the anglicised version to the native French ones in this PR?

Reply:
Done.